src/plone/rest/negotiation.py

```diff
@@ -44,7 +42,7 @@ def register_service(method, media_type):
     """Register a service for the given request method and media type and
        return it's service id.
     """
-    service_id = u'{}_{}_{}'.format(NAME_PREFIX, media_type[0], media_type[1])
+    service_id = u'{}_{}_{}_'.format(method, media_type[0], media_type[1])
```

Comment:
It's there to have consistent view names following the pattern `{METHOD}_{TYPE}_{SUBTYPE}_{SERVICENAME}`. Thus unnamed services have a trailing `_`, e.g. `GET_application_json_`. Would also work without that of course.